### ISSUE TITLE
Audio input

### ISSUE THREAD

**Issue by pmatos** [jbclements/portaudio#12]
I have been looking for a way to get an audio stream into racket. Portaudio seems to be able to do that:
http://files.portaudio.com/docs/v19-doxydocs/open_default_stream.html

However, the bindings don't seem to allow anything like this. Is this still in the works?

**Comment by jbclements** [comment]
Hmm.. I'm not sure I understand the question. This library does support streaming output, and recording fixed-length inputs. Are you looking for the ability to record streaming audio input?

**Comment by pmatos** [comment]
Yes, I am looking for the ability to record streaming input. Do you know what's involved in something like this?

Guess we need to extend the bindings, right?

**Comment by jbclements** [comment]
Yes!
